src/ScheduleService.Models.Dto/Requests/Group/CreateGroupRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace LT.DigitalOffice.ScheduleService.Models.Dto.Requests.Group;
+
+public class CreateGroupRequest
```

Comment:
I guess it needs to have list of user ids(in case group is a group of users. Correct me if I'm wrong)